test/testUtils.js

```diff
@@ -850,3 +857,142 @@ export const chartsDataFilterResponse = (overrides = {}) => ({
   lastModified: '',
   ...overrides,
 })
+export const assessmentDayDataControllerMetadata =
```

Comment:
why do these functions go in the test utils file? Can they not be defined in the tests themselves?

Reply:
They were defined in the tests themselves and I moved them to clean up the file and make it less noisy, because I was having a hard time finding my place in the file when writing tests

Reply:
we should be able to leave them in the test files but remove a lot of the attributes. The value of the fixtures is that they will create default values for all the attributes so we only need to define the ones we care about.

Reply:
I've made the updates and some of the variables need to be reused and they were defined at every test, which made it hard to read. To your point it is a combination of both the properties and reusing the variables. 
[0872fb6](https://github.com/AMP-SCZ/dpdash/pull/726/commits/0872fb6ba250403a2178df163c52f903ce4d1518)